### PR TITLE
add scm element in pom

### PR DESCRIPTION
**Pull request by eyalgo**
Added scm to pom, as maven central requests.

### CODE REVIEW

**Comment by eyalgo** [comment]
I did some modification in the pom files.
(source and target version were not aligned).

added license to parent pom

**Comment by eyalgo** [comment]
hi,
can we merge the pull request?
I will then go forward and deploy the project.

File: pom.xml [hunk 1]
```diff
@@ -71,7 +86,7 @@
 					<version>3.2</version>
 					<configuration>
 						<source>1.8</source>
-						<target>1.7</target>
+						<target>1.8</target>
```
Comment (hakandilek) [inline pom.xml]: Only this one is making me nervous. I think we should stick with Java 7 as a target as long as we don't need Java 8.

**Comment by eyalgo** [comment]
set to 1.7

**Pull request merged**